packages/larva/lib/writeJson.js

```diff
@@ -18,27 +18,42 @@ module.exports = function writeJson( patternConfig, fromLarva = false ) {
 
 		const startPath = sourceDirectory + '/modules/' + moduleName;
 		const variants = getPatternVariants( startPath );
-		
+
 		variants.forEach( ( variant ) => {
 
-			moduleData = getPatternData( sourceDirectory, {
+			const moduleData = getPatternData( sourceDirectory, {
 				type: 'modules',
 				name: moduleName,
 				variant: variant
 			} );
 
-			jsonDestPath = path.resolve( patternConfig.projectPatternsDir, '../../build/json/modules/' + moduleName + '.' + variant + '.json' );
-
-				// If JSON data and module prototype are the same, pass, otherwise write the data to file
-				if ( fs.existsSync( jsonDestPath ) && JSON.stringify( require( jsonDestPath ) ) === JSON.stringify( moduleData ) ) {
-					console.log( chalk.grey( `No updates in ${moduleName}.${variant}` ) );
-				} else {
-					if ( undefined !== moduleData ) {
-						writeJsonToFile( jsonDestPath, moduleData );
-						console.log( chalk.green.bold( `Wrote JSON for ${moduleName}.${variant}` ) );
-					}
+			const jsonDestPath = path.resolve( patternConfig.projectPatternsDir, '../../build/json/modules/' + moduleName + '.' + variant + '.json' );
+
+			// If JSON data and module prototype are the same, pass, otherwise write the data to file
+			let hasChanged = true;
+			try {
+				if (
+					fs.existsSync( jsonDestPath ) &&
+						JSON.stringify( require( jsonDestPath ) ) ===
+							JSON.stringify( moduleData )
+				) {
+					hasChanged = false;
 				}
+			} catch ( error ) {
+				console.warn( error );
+			}
+
+			if ( hasChanged ) {
+				writeJsonToFile( jsonDestPath, moduleData );
+				console.log(
```

Comment:
Same for the console logs - other logs throughout the project are on a single line. I do think a max-len linter config is a good idea here so that this can be consistent.

Reply:
updated